model/task/task.go

```diff
@@ -561,6 +562,92 @@ func (t *Task) MarkEnd(finishTime time.Time, detail *apimodels.TaskEndDetail) er
 
 }
 
+func (t *Task) UpdateDisplayTask() error {
+	if !t.DisplayOnly {
+		return fmt.Errorf("%s is not a display task", t.Id)
+	}
+
+	statuses := []string{}
+	catcher := grip.NewBasicCatcher()
+	var timeTaken time.Duration
+	for _, taskId := range t.ExecutionTasks {
+		execTask, err := FindOne(ById(taskId))
```

Comment:
should we try and do an $in query to avoid the extra round trips?

Reply:
done